content/zh/blog/cncf-sandbox-announcement/index.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "KubeEdge has been accepted into the CNCF Sandbox"
```

Comment:
The title is inconsistent with the content and English article should not be in `content/zh`.

Reply:
Updated. Please a take a look.